pyteal/compiler/compiler.py

```diff
@@ -109,9 +117,9 @@ def verifyOpsForMode(teal: List[TealComponent], mode: Mode):
 def compileSubroutine(
     ast: Expr,
     options: CompileOptions,
-    subroutineMapping: Dict[Optional[SubroutineDefinition], List[TealComponent]],
     subroutineGraph: Dict[SubroutineDefinition, Set[SubroutineDefinition]],
     subroutineBlocks: Dict[Optional[SubroutineDefinition], TealBlock],
```

Comment:
nit: would be great if you rename it `subroutine_start_blocks` (if I understand it correctly, this maps `SubroutineDefinition` to the `TealBlock` it starts?)